requirements-optional.txt

```diff
@@ -0,0 +1,2 @@
+yapf<0.33.*
```

Comment:
Maybe mention the splitup of requirements and optional requirements in the README?

Reply:
good idea